backend/serializers.py

```diff
@@ -1,8 +1,15 @@
 from rest_framework import serializers
-from backend.models import Post 
+from backend.models import Post, Comment
 
 # Post Serializer 
 class PostSerializer(serializers.ModelSerializer): 
     class Meta: 
         model = Post 
-        fields = '__all__'
\ No newline at end of file
+        fields = '__all__'
+        
+        
+# Comment Serializer 
+class CommentSerializer(serializers.ModelSerializer): 
+    class Meta: 
+        model = Comment
+        fields = ['uuid', 'commentType', 'author', 'post', 'comment', 'contentType', 'published']
```

Comment:
can you specify the read only fields? the way this is setup will allow writing published, uuid, etc. 
put those under read_only_fields

Reply:
The comment model changed completely so I had to delete the migration and create again. It was giving errors because the tables had conflict, didn't have any choice there.

Reply:
delete your migrations, delete your database, pull migrations from master, migrate database, makemigrations, then it should create a migration with the diff rather than this. you do have a choice, the choice is to fix it on your end before pushing it to master which will create a problem for everyone. please do not push this without fixing it.